### ISSUE TITLE
P1: JWT upload button isn’t actionable from keyboard

### ISSUE THREAD

**Issue by asimpson** [grafana/grafana#56563]
The link that opens the file picker is set to `tabindex=0` which makes it not accessible via keyboard.
![image](https://user-images.githubusercontent.com/1048831/194577006-1742e2d3-79b2-41ba-9508-362287247b51.png)

**Comment by asimpson** [comment]
- moved to https://github.com/grafana/grafana-google-sdk-react/issues/8
